guest-agents/metal-agent/pkg.yaml

```diff
@@ -24,6 +24,11 @@ steps:
         cp -r /rootfs/ /extensions-validator-rootfs/rootfs
         cp /pkg/manifest.yaml /extensions-validator-rootfs/manifest.yaml
         /extensions-validator validate --rootfs=/extensions-validator-rootfs --pkg-name="${PKG_NAME}"
+    sbom:
+      outputPath: /rootfs/usr/local/share/spdx/metal-agent.spdx.json
+      version: {{ .VERSION }}
+      licenses:
+        - MPL-2.0
```

Comment:
This is a Go project: https://github.com/siderolabs/talos-metal-agent

Maybe it's worth generating an SBOM in its Dockerfile, just like it's done for Talos. Not sure. @smira wdyt?